modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -5,46 +5,185 @@
 [id="additional-install-config-parameters_{context}"]
 = Additional `install-config` parameters
 
-This topic describes the required parameters, the `hosts` parameter, and the `bmc` `address` parameter for the `install-config.yaml` file.
+See the following tables for the required parameters, the `hosts` parameter,
+and the `bmc` parameter for the `install-config.yaml` file.
 
 .Required parameters
-
 |===
 |Parameters |Default |Description
-| `provisioningNetworkInterface` |  | The name of the network interface on control plane nodes connected to the
-provisioning network. ({product-title} 4.4 only)
-| `hosts` |  | Details about bare metal hosts to use to build the cluster.
+
+
+| [[basedomain]] `baseDomain`
+|
+| The domain name for the cluster. For example, `example.com`.
+
+
+a|[[metadataname]]
+----
+metadata:
+    name:
+----
+|
+|The name to be given to the {product-title} cluster. For example, `openshift`.
+
+
+a|[[machinecidr]]
+----
+networking:
+    machineCIDR:
+----
+|
+|The public CIDR (Classless Inter-Domain Routing) of the external network. For example, `10.0.0.0/24` or `2620:52:0:1302::/64`.
+
+
+a|[[workername]]
+----
+compute:
+  - name: worker
+----
+|
+|The {product-title} cluster requires a name be provided for worker (or compute) nodes even if there are zero nodes.
+
+
+a|[[computereplicas]]
+----
+compute:
+    replicas: 2
+----
+|
+|Replicas sets the number of worker (or compute) nodes in the {product-title} cluster.
+
+
+a|[[controlplanename]]
+----
+controlPlane:
+    name: master
+----
+|
+|The {product-title} cluster requires a name for control plane (master) nodes.
+
+
+a|[[controlplanereplicas]]
+----
+controlPlane:
+    replicas: 3
+----
+|
+|Replicas sets the number of control plane (master) nodes included as part of the {product-title} cluster.
```

Comment:
drop master here?

Reply:
Need it there, because it's still named master in the software. I've informed Roger about this.